Client/game_sa/CPoolsSA.cpp

```diff
@@ -410,6 +411,14 @@ void CPoolsSA::RemoveBuilding(CBuilding* pBuilding)
     // Remove building from world
     pGame->GetWorld()->Remove(pInterface, CBuildingPool_Destructor);
 
+    // Remove building from cover list
+    CPtrNodeSingleListSAInterface<CBuildingSAInterface>* coverList = reinterpret_cast<CPtrNodeSingleListSAInterface<CBuildingSAInterface>*>(0xC1A2B8);
+    coverList->RemoveItem(pInterface);
+
+    // Remove plant
+    typedef CEntitySAInterface*(__cdecl * CPlantColEntEntry_Remove)(CEntitySAInterface * item);
```

Comment:
Why not use `using` instead of `typedef`?

Reply:
In this case, there is no difference. Do i miss something?

Reply:
> In this case, there is no difference. Do i miss something?

not really a difference. Just that in C++ its better to use `using` than `typedef`

Reply:
Would be better to provide actual details on the differences
> A typedef-name can also be introduced by an alias-declaration. The identifier following the using keyword becomes a typedef-name and the optional attribute-specifier-seq following the identifier appertains to that typedef-name. It has the same semantics as if it were introduced by the typedef specifier. In particular, it does not define a new type and it shall not appear in the type-id.

> The alias declaration is compatible with templates, whereas the C style typedef is not.

Reply:
> Would be better to provide actual details on the differences
> 
> > A typedef-name can also be introduced by an alias-declaration. The identifier following the using keyword becomes a typedef-name and the optional attribute-specifier-seq following the identifier appertains to that typedef-name. It has the same semantics as if it were introduced by the typedef specifier. In particular, it does not define a new type and it shall not appear in the type-id.
> 
> > The alias declaration is compatible with templates, whereas the C style typedef is not.

`using` is the same as `typedef` with the exception of templates. Doesn't really matter. `typedef` is a C thing. We have `using` in C++ so let's start using `using`

Reply:
Yeah, I didn't make an opinion either way, just providing the relevant details so that others can accurately come to a conclusion. It's better than simply saying "there's not really a difference" when there are actual differences.